frontend/src/components/associations/marketplace/History.tsx

```diff
@@ -82,55 +84,113 @@ const NoTransactionMessage = () => (
         status = <span className="tag tag-green">Délivrée</span>;
     }
-    if (transaction.status === "CANCELLED") {
+    if (transaction.status === TransactionStatus.Cancelled) {
         status = <span className="tag tag-red">Annulée</span>;
     }
-    if (transaction.status === "REFUNED") {
+    if (transaction.status === TransactionStatus.Refunded) {
         status = <span className="tag tag-yellow">Remboursée</span>;
     }
+    if (transaction.status === TransactionStatus.Rejected) {
+        status = <span className="tag tag-red">Refusée</span>;
+    }
 
     return (
         <tr>
             <td>
-                <strong>{transaction.product.name}</strong>
+                <strong>{transaction.product.name}</strong>{" "}
+                <span className="text-muted">x{transaction.quantity}</span>
             </td>
-            <td className="text-center text-muted d-none d-md-table-cell text-nowrap">
-                Quantité : {transaction.quantity}
+            <td className="text-center">
+                <strong>{transaction.value}€</strong>
             </td>
             <td className="text-right">
-                <strong>{transaction.value}€</strong>
+                {formatDate(new Date(transaction.date))}
             </td>
             <td className="text-right">{status}</td>
         </tr>
     );
 };
 
-const RefundTransactionLine = ({ transaction }) => {
+// Funding informations
+const FundingHistory = ({ marketplaceId, user }) => {
+    return (
+        <Pagination
+            apiKey={[
+                "marketplace.funding.list",
+                marketplaceId,
+                { page_size: 10, user: user?.id },
+            ]}
+            apiMethod={api.fundings.list}
+            render={(fundings, paginationControl) => (
+                <Container>
+                    <PageTitle>Mes dépôts</PageTitle>
+
+                    {fundings.length !== 0 ? (
+                        <>
+                            <FundingTable fundings={fundings} />
+                            {paginationControl}
+                        </>
+                    ) : (
+                        <NoFundingMessage />
+                    )}
+                </Container>
+            )}
+        />
+    );
+};
+
+const FundingTable = ({ fundings }) => {
+    return (
+        <Card>
+            <table className="table card-table table-vcenter">
+                <tbody>
+                    {fundings.map((funding, index) => (
+                        <FundingLine key={index} funding={funding} />
+                    ))}
+                </tbody>
+            </table>
+        </Card>
+    );
+};
+
+const NoFundingMessage = () => (
+    <Card>
+        <Card.Body>
+            <p className="text-center">
+                Vous n'avez jamais rempli votre compte jusqu'à maintenant
+            </p>
+        </Card.Body>
+    </Card>
+);
+
+const FundingLine = ({ funding }) => {
```

Comment:
Please consider renaming to `FundingRow` which is more explicit when dealing with tables

Reply:
or idiomatic